### DISCUSSION TITLE
Github Pro expiration

### DISCUSSION THREAD

**Discussion by Xyndra** [community/community#128941] [GitHub Education] [answered]
### Hi everyone,

![image](https://github.com/community/community/assets/71836523/0751af64-dd90-4f17-bb7b-96504d0073f9)

I am still in school and don't know what the problem is. I thought GitHub Pro was free as long as you are a student? Could someone please help me, I don't want to lose my GitHub Copilot etc. Do I have to add a payment method?

**Comment by davevad93** [comment, marked as answer]
Hi @Xyndra ,

The student validation lasts for 2 years, after which you will need to revalidate your student status by re-applying on the [GitHub Education application page](https://education.github.com/discount_requests/application).

**Reply by Xyndra** [reply]
oh, I thought it was auto-renewing since it lasted longer than one year, thanks

**Reply by davevad93** [reply]
You're welcome! 👍
